Tasks/UseNodeV1/installer.ts

```diff
@@ -173,27 +170,27 @@ async function acquireNode(version: string): Promise<string> {
 // and lib file in a folder, not zipped.
 async function acquireNodeFromFallbackLocation(version: string): Promise<string> {
     // Create temporary folder to download in to
-    let tempDownloadFolder: string = 'temp_' + Math.floor(Math.random() * 2000000000);
-    let tempDir: string = path.join(taskLib.getVariable('agent.tempDirectory'), tempDownloadFolder);
+    const tempDownloadFolder: string = 'temp_' + Math.floor(Math.random() * 2e9);
+    const tempDir: string = path.join(taskLib.getVariable('agent.tempDirectory'), tempDownloadFolder);
     taskLib.mkdirP(tempDir);
+
     let exeUrl: string;
     let libUrl: string;
     try {
-        exeUrl = `https://nodejs.org/dist/v${version}/win-${os.arch()}/node.exe`;
-        libUrl = `https://nodejs.org/dist/v${version}/win-${os.arch()}/node.lib`;
+        exeUrl = `https://nodejs.org/dist/v${version}/win-${osArch}/node.exe`;
```

Comment:
Should the change be made here too?

Reply:
Actually, maybe just move the change up to where we define osArch in line 8? That way it propogates to error messages too. So it could be

```
const arch = os.arch(); 
const osArch = (arch === 'ia32') ? 'x86' : arch; 
```

Reply:
You're right, 'x86' should be used in all cases here.